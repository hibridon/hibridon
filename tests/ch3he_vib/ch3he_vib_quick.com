inp=ch3he_vib_quick.inp
job=vibq
run
intcrs
