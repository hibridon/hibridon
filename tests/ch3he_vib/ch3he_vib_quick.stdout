
 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V xdate

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
SYS=Debian bullseye/sid ( 5.4.0-80-generic); MACH=
CC=; FC=



 --------------------------------------------------------------------------
 Hibridon>  Hibridon>  Hibridon>  ** J =    0 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 11:13:27 2021
 ** J =    1 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 11:13:28 2021
 ** J =    2 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 11:13:28 2021
 ** J =    3 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 11:13:29 2021
 ** J =    0 JLPAR =-1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 11:13:30 2021
 ** J =    1 JLPAR =-1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 11:13:30 2021
 ** J =    2 JLPAR =-1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 11:13:30 2021
 ** J =    3 JLPAR =-1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 11:13:31 2021
 ===============================================================================
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:   148
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    NaN %
      CURRENT DATE:  Aug 12 11:13:31 2021
 ===============================================================================
 Hibridon> 
 INTEGRAL CROSS SECTIONS CALCULATED FROM S-MATRICES ON FILE Vibq1.smt                               
 WRITTEN:   Aug 12 11:13:27 2021
 LABEL:     CH3-He Vib QMa                                  
 JTOT_MAX:    3
 IEN= 1 RMU=   3.1606 E=   642.50 JTOT-1=  0 JTOT-2=   3 JTOT-D=  1

 READ S-MATRIX FOR JTOT=  0  JLPAR= 1
 READ S-MATRIX FOR JTOT=  0  JLPAR=-1

 LEVEL LIST FOR INTEGRAL CROSS SECTIONS              
   N     J   INDEX  EINT(cm-1)
   1     0     0      0.000
   2     1   100    624.969
   3     2     0     57.467
   4     3    -3     71.418
   5     4     3    148.042
   6     4     0    191.558
   7     5    -3    243.820
   8     6     6    228.206
   9     6     3    358.755
  10     6     0    402.271
  11     7    -6    362.297
  12     7    -3    492.846
  13     8     6    515.543
  14     9    -9    470.364

 CC INTEGRAL CROSS SECTIONS

 ** COLUMN HEADINGS ARE FINAL STATES, ROW HEADINGS ARE INITIAL STATES **

          1           2           3           4           5           6
   1   6.5366E-01  1.2921E-05  4.9712E-02  6.9477E-02  1.9383E-05  1.7650E-02
   2   1.5786E-04  2.7734E+01  2.7032E-04  3.2322E-04  6.3411E-04  1.1039E-03
   3   1.0919E-02  4.8601E-06  5.3592E-01  4.7011E-02  5.5963E-02  3.9663E-02
   4   1.1167E-02  4.2523E-06  3.4400E-02  4.9147E-01  3.2443E-02  2.4328E-02
   5   2.7985E-06  7.4940E-06  3.6786E-02  2.9144E-02  4.1304E-01  3.2082E-02
   6   2.7941E-03  1.4304E-05  2.8587E-02  2.3963E-02  3.5178E-02  4.3971E-01
   7   9.5639E-03  1.5406E-05  8.6848E-03  3.1388E-02  2.0809E-02  6.2200E-02
   8   8.3407E-03  1.0619E-06  3.2929E-02  4.7183E-02  2.9999E-02  2.4105E-02
   9   5.3410E-07  3.1962E-06  9.5283E-03  6.9373E-03  1.8728E-02  1.7252E-02
  10   1.2290E-03  2.6206E-05  6.9651E-03  7.4174E-03  9.8750E-03  2.6144E-02
  11   1.6052E-07  3.3146E-06  1.1174E-02  8.4336E-03  2.7221E-02  1.8384E-02
  12   5.8802E-04  3.0311E-05  3.2152E-03  3.5609E-03  5.9118E-03  9.0611E-03
  13   2.9051E-04  2.6256E-06  3.6365E-04  1.8194E-03  1.1463E-04  3.8017E-03
  14   7.2609E-04  6.7338E-08  3.0382E-03  6.7809E-03  7.0321E-03  2.0130E-03

          7           8           9          10          11          12
   1   6.5280E-02  6.9916E-02  3.0663E-06  5.9736E-03  1.0501E-06  2.0545E-03
   2   1.2847E-03  1.0874E-04  2.2418E-04  1.5561E-03  2.6489E-04  1.2938E-03
   3   1.3020E-02  6.0628E-02  1.2015E-02  7.4361E-03  1.6056E-02  2.4674E-03
   4   3.4434E-02  6.3568E-02  6.4013E-03  5.7946E-03  8.8671E-03  1.9996E-03
   5   2.0506E-02  3.6307E-02  1.5523E-02  6.9300E-03  2.5710E-02  2.9822E-03
   6   6.7212E-02  3.1988E-02  1.5680E-02  2.0118E-02  1.9039E-02  5.0118E-03
   7   3.8871E-01  3.4499E-02  2.5454E-02  1.4380E-02  2.8892E-02  8.5699E-03
   8   2.8091E-02  3.0484E-01  6.1917E-03  4.4320E-03  1.3097E-02  1.6688E-03
   9   3.0262E-02  9.0405E-03  4.7966E-01  8.6391E-02  5.6379E-02  8.5262E-03
  10   2.0193E-02  7.6434E-03  1.0204E-01  6.3612E-01  1.4982E-02  9.0452E-02
  11   3.0146E-02  1.6782E-02  4.9479E-02  1.1132E-02  4.8649E-01  8.6351E-03
  12   1.6742E-02  4.0039E-03  1.4010E-02  1.2584E-01  1.6168E-02  9.6291E-01
  13   1.0178E-02  5.0582E-03  2.3505E-02  1.3015E-02  1.2614E-02  7.1344E-02
  14   4.6334E-03  3.2792E-02  3.4610E-03  5.8329E-04  2.3876E-02  1.1778E-03

         13          14
   1   9.7588E-04  3.6961E-03
   2   1.0775E-04  4.1876E-06
   3   2.6831E-04  3.3969E-03
   4   9.8226E-04  5.5478E-03
   5   5.5595E-05  5.1682E-03
   6   2.0217E-03  1.6222E-03
   7   5.0088E-03  3.4555E-03
   8   2.0270E-03  1.9914E-02
   9   1.3753E-02  3.0687E-03
  10   8.9945E-03  6.1086E-04
  11   6.4771E-03  1.8579E-02
  12   6.8594E-02  1.7160E-03
  13   1.1338E+00  2.5791E-02
  14   1.7020E-02  7.3118E-01

 ** INTCRS FINISHED. ELAPSED TIME:  00:00:00   CPU TIME:  00:00:00  **
 Hibridon> 
